tsconfig.compodoc.json

```diff
@@ -20,6 +20,9 @@
       "@spartacus/cds": ["projects/cds/public_api"],
       "@spartacus/schematics": ["projects/schematics/src/public_api"],
       "@spartacus/my-account": ["feature-libs/my-account/public_api"],
+      "@spartacus/my-account/organization": [
+        "feature-libs/my-account/organization/public_api"
+      ],
```

Comment:
Shouldn't it also contain the same additional entry points as the main `tsconfig.json`? 
FYI @dunqan

Reply:
Not verified how it affects compodocs, but yes, looks like it should have it

Reply:
Synced with root `tsconfig.json`